### PR TITLE
Completed 2 more sorries and fixed some bracketing

### PR DESCRIPTION
**Pull request by nimarasekh**

### CODE REVIEW

**Comment by dagurtomas** [comment]
Can you fix the merge conflicts on your branch?

**Comment by nimarasekh** [comment]
It should be good now, but feel free to check. Some of these conflicts made no sense, so either I did something weird or something got GitHub confused.

File: LeanCondensed/Projects/AB.lean [hunk 1]
```diff
@@ -32,11 +32,11 @@ example : I ⥤ ShortComplex A ≌ ShortComplex (I ⥤ A) :=
 
 lemma forall_exact_iff_functorEquivalence_exact (F : I ⥤ ShortComplex A) : (∀ i, (F.obj i).Exact) ↔
     ((functorEquivalence I A).inverse.obj F).Exact := by
-  sorry
+    sorry
 
 class HasExactLimitsOfShape : Prop where
   hasLimitsOfShape : HasLimitsOfShape I A := by infer_instance
-  exact_limit (F : I ⥤ ShortComplex A) : (∀ i, (F.obj i).ShortExact) → (limit F).ShortExact
+  exact_limit : ∀ (F : I ⥤ ShortComplex A), (∀ i, (F.obj i).ShortExact) → (limit F).ShortExact
```
Comment (dagurtomas) [inline LeanCondensed/Projects/AB.lean]:
```suggestion
  exact_limit (F : I ⥤ ShortComplex A) :  (∀ i, (F.obj i).ShortExact) → (limit F).ShortExact
```

File: LeanCondensed/Projects/AB.lean [hunk 2]
```diff
@@ -50,14 +50,27 @@ lemma hasExactLimitsOfShape_iff_lim_rightExact [HasLimitsOfShape I A] :
...
+          have eq := IsLimit.conePointUniqueUpToIso (isLimitLimitCone F) (limit.isLimit F)
+          apply shortExact_of_iso eq
+          exact (h F hh)
+         · exact B
+
 
 class HasExactColimitsOfShape : Prop where
   hasColimitsOfShape : HasColimitsOfShape I A := by infer_instance
-  exact_colimit (F : I ⥤ ShortComplex A) : (∀ i, (F.obj i).ShortExact) → (colimit F).ShortExact
+  exact_colimit : ∀ (F : I ⥤ ShortComplex A), ((∀ i, (F.obj i).ShortExact) → (colimit F).ShortExact)
```
Comment (dagurtomas) [inline LeanCondensed/Projects/AB.lean]:
```suggestion
  exact_colimit (F : I ⥤ ShortComplex A) : ((∀ i, (F.obj i).ShortExact) → (colimit F).ShortExact)
```

File: LeanCondensed/Projects/AB.lean [hunk 3]
```diff
@@ -114,6 +139,7 @@ lemma left_exact_of_left_exact [HasLimitsOfShape I A] (F : I ⥤ ShortComplex A)
     Mono (ShortComplex.limitCone F).pt.f ∧ (ShortComplex.limitCone F).pt.Exact := by
   sorry
 
+-- NR: Made this one up, think it should be what we want.
```
Comment (dagurtomas) [inline LeanCondensed/Projects/AB.lean]:
```suggestion
```

File: LeanCondensed/Projects/AB.lean [hunk 4]
```diff
@@ -150,7 +176,8 @@ lemma abStar_iff_preserves_epi [HasLimitsOfShape I A] :
     have := h F hh
     exact this.epi_g
 
-lemma ab_iff_preserves_mono [HasColimitsOfShape I A] :
+-- Stating and proving the converse of this lemma should be easy
+lemma ab_of_preserves_mono [HasColimitsOfShape I A] :
```
Comment (dagurtomas) [inline LeanCondensed/Projects/AB.lean]:
```suggestion
lemma ab_iff_preserves_mono [HasColimitsOfShape I A] :
```

File: LeanCondensed/Projects/AB.lean [hunk 5]
```diff
@@ -186,10 +213,7 @@ namespace LightCondensed
 
 variable (R : Type u) [Ring R]
 
-instance : sequentialAB4star (LightCondMod.{u} R) := by
-  apply sequentialAB4star_of_epi_limit_of_epi
-  intros
-  exact LightCondensed.epi_limit_of_epi _
+instance : sequentialAB4star (LightCondMod.{u} R) := by sorry
```
Comment (dagurtomas) [inline LeanCondensed/Projects/AB.lean]:
```suggestion
instance : sequentialAB4star (LightCondMod.{u} R) := by
  apply sequentialAB4star_of_epi_limit_of_epi
  intros
  exact LightCondensed.epi_limit_of_epi _
```

**Review by dagurtomas — APPROVED** [review]

File: LeanCondensed/Projects/AB.lean [hunk 6]
```diff
@@ -32,11 +32,11 @@ example : I ⥤ ShortComplex A ≌ ShortComplex (I ⥤ A) :=
 
 lemma forall_exact_iff_functorEquivalence_exact (F : I ⥤ ShortComplex A) : (∀ i, (F.obj i).Exact) ↔
     ((functorEquivalence I A).inverse.obj F).Exact := by
-  sorry
+    sorry
```
Comment (dagurtomas) [inline LeanCondensed/Projects/AB.lean]:
```suggestion
  sorry
```

File: LeanCondensed/Projects/AB.lean [hunk 7]
```diff
@@ -36,7 +36,7 @@ lemma forall_exact_iff_functorEquivalence_exact (F : I ⥤ ShortComplex A) : (
 
 class HasExactLimitsOfShape : Prop where
   hasLimitsOfShape : HasLimitsOfShape I A := by infer_instance
-  exact_limit (F : I ⥤ ShortComplex A) : (∀ i, (F.obj i).ShortExact) → (limit F).ShortExact
+  exact_limit (F : I ⥤ ShortComplex A) :  (∀ i, (F.obj i).ShortExact) → (limit F).ShortExact
```
Comment (dagurtomas) [inline LeanCondensed/Projects/AB.lean]:
```suggestion
  exact_limit (F : I ⥤ ShortComplex A) : (∀ i, (F.obj i).ShortExact) → (limit F).ShortExact
```

**Pull request merged**